[2024-05-23T17:12:10.472+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-23T17:12:10.484+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-23T17:12:08.349692+00:00 [queued]>
[2024-05-23T17:12:10.488+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-23T17:12:08.349692+00:00 [queued]>
[2024-05-23T17:12:10.488+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-23T17:12:10.498+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-23 17:12:08.349692+00:00
[2024-05-23T17:12:10.503+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=92) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-23T17:12:10.504+0000] {standard_task_runner.py:63} INFO - Started process 96 to run task
[2024-05-23T17:12:10.504+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'games', 'create_db', 'manual__2024-05-23T17:12:08.349692+00:00', '--job-id', '709', '--raw', '--subdir', 'DAGS_FOLDER/get_main_games.py', '--cfg-path', '/tmp/tmpmisej668']
[2024-05-23T17:12:10.506+0000] {standard_task_runner.py:91} INFO - Job 709: Subtask create_db
[2024-05-23T17:12:10.530+0000] {task_command.py:426} INFO - Running <TaskInstance: games.create_db manual__2024-05-23T17:12:08.349692+00:00 [running]> on host 1af7b740efc2
[2024-05-23T17:12:10.619+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T17:12:08.349692+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-23T17:12:08.349692+00:00'
[2024-05-23T17:12:10.620+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-23T17:12:10.640+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2024-05-23T17:12:12.607+0000] {logging_mixin.py:188} INFO - Таблица 'games' удалена.
[2024-05-23T17:12:13.016+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-23T17:12:13.017+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-23T17:12:13.024+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=games, task_id=create_db, execution_date=20240523T171208, start_date=20240523T171210, end_date=20240523T171213
[2024-05-23T17:12:13.045+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-23T17:12:13.061+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-23T17:12:13.061+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
